src/capitalsFirst.js

```diff
@@ -18,7 +18,20 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let UpperStrArr = [];
```

Comment:
variable names should be lowerCamelCase. Which means: UpperStrArr => upperStrArr.
Same below.